### DISCUSSION TITLE
great praise for the new editor!

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretalx/pretalx#1578] [General]
Hey People,
we want to thank you a lot for the new editor.

We really like the following features:

- in the timeline on the left side there is also the weekday written
- you can quickly jump to a date and hide rooms.
- the vertical magnetic docking of talks when moving them
- Times in which nothing happens are hidden
- More info about the talks visible in the editor

Thanks a lot for all this!!

Since we have 13 rooms in parallel, however, this new "size" is a disadvantage, even on a QHD display you can't get them all on at the same screen in a useful way, and for us it's important that we can see which talks are in parallel when planning.
Witht the old editor, we were able to edit all of them on a full HD monitor.

Could the menu and the unallocated talks be made hideable (einklappbar)?

I think we will try a 4K Monitor next ;)

**Comment by rixx** [comment]
You can already hide the menu by clicking the ≡ button at the top left. Other than that, I'd recommend hiding rooms occasionally, or zooming out? I don't think we'll support showing huge amounts of rooms at the same time for now, as most events don't want that – they'd rather focus on a few rooms (by track, or building, or floor) at a time. Sorry – just too much dev effort for a rather niche use case (although, as per usual, happy to look at pull requests).
